pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -113,7 +192,179 @@ def load(fnames, tag=None, inst_id=None, file_cadence=dt.timedelta(days=1),
+    drop_meta_labels : list or NoneType
+        List of variable metadata labels that should be dropped. Applied
+        to metadata as loaded from the file. (default=None)
+
+    Returns
+    -------
+    data : xarray.Dataset
+        Class holding file data
+    meta : pysat.Meta
+        Object containing metadata such as column names and units
+
+    Note
+    ----
+    This routine is intended to be used by pysat instrument modules supporting
+    a particular NASA CDAWeb dataset.
+
+    Examples
+    --------
+    ::
+
+        # within the new instrument module, at the top level define
+        # a new variable named load, and set it equal to this load method
+        # code below taken from cnofs_ivm.py.
+
+        # support load routine
+        # use the default CDAWeb method
+        load = functools.partial(cdw.load, pandas_format=False)
+
+    """
+
+    # Load data from any files provided
+    if len(fnames) <= 0:
+        return xr.Dataset()
+    else:
+        # Using cdflib wrapper to load the CDF and format data and
+        # metadata for pysat using some assumptions. Depending upon your needs
+        # the resulting pandas DataFrame may need modification
+        ldata = []
+        for lfname in fnames:
+            temp_data = cdflib.cdf_to_xarray(lfname, to_datetime=True)
+            ldata.append(temp_data)
+        # Combine individual files together
+        if len(ldata) > 0:
+            data = xr.combine_by_coords(ldata)
+
+    all_vars = io.xarray_all_vars(data)
+
+    # Convert output epoch name to 'time' for pysat consistency
+    if epoch_name != 'time':
+        if 'time' not in all_vars:
+            if epoch_name in data.dims:
+                data = data.rename({epoch_name: 'time'})
+            elif epoch_name in all_vars:
+                data = data.rename({epoch_name: 'time'})
+                wstr = ''.join(['Epoch label: "', epoch_name, '"',
+                                ' is not a dimension.'])
+                pysat.logger.warning(wstr)
+            else:
+                estr = ''.join(['Epoch label: "', epoch_name, '"',
+                                ' not found in loaded data, ',
+                                repr(all_vars)])
+                raise ValueError(estr)
+
+        epoch_name = 'time'
+
+    all_vars = io.xarray_all_vars(data)
+
+    meta = pysat.Meta(labels=labels)
+
+    full_mdict = {}
+
+    if meta_translation is None:
+        # Assign default translation using `meta`
+        meta_translation = {}
+
+    # Drop metadata labels initialization
+    if drop_meta_labels is None:
+        drop_meta_labels = []
+    else:
+        drop_meta_labels = pysat.utils.listify(drop_meta_labels)
+
+    for key in all_vars:
+        meta_dict = {}
+        for nc_key in data[key].attrs.keys():
+            meta_dict[nc_key] = data[key].attrs[nc_key]
+        full_mdict[key] = meta_dict
+        data[key].attrs = {}
+
+    for data_attr in data.attrs.keys():
```

Comment:
I believe this gets ahead of the `use_header` keyword argument. I believe pysat moves all non-standard meta attributes to the header if the user enables the flag, otherwise they go to the Instrument object.

Reply:
Just saw the same line in load_netcdf_pandas, so.... I probably need to recheck the whole header thing 😸 
```
            # Build a dictionary with all global ncattrs and add those
            # attributes to a pysat.MetaHeader object.
            for ncattr in data.ncattrs():
                setattr(meta.header, ncattr, data.getncattr(ncattr))
```